# an empty map will use the currently loaded map
string map_name

string name

# Update fields
string description
int32 marker_type
geometry_msgs/Pose pose
---
bool success
string message